[9.0] Add missing index

sebastienbeau opened the pull request:
In case of big table adding the index on mts_mto_procurement_id
improve the speed to read of mts_mto_procurement_ids

same PR as https://github.com/OCA/stock-logistics-warehouse/pull/775

rousseldenis commented:
/ocabot merge patch

bealdav commented:
blocked ?

rousseldenis commented:
@sebastienbeau Could you update module version as I cannot use ocabot to merge (old branch so no runbot) ?

sebastienbeau committed:
IMP stk_mts_mto_rule: Add missing index

    In case of big table adding the index on mts_mto_procurement_id
    improve the speed to read of mts_mto_procurement_ids

bealdav commented:
Done
